src/components/PersistentDrawerLeft/PersistentDrawerLeft.css

```diff
@@ -1,6 +1,39 @@
+
+/*chrome and safari*/
 .MuiDrawer-paper::-webkit-scrollbar {
-  display: none;
+  width: 0.5em;
+  background-color: #000000;
+}
+
+.MuiDrawer-paper::-webkit-scrollbar-track {
+  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+  background: rgb(0,0,0);
+  background-clip: content-box; 
+}
+
+.MuiDrawer-paper::-webkit-scrollbar-thumb {
+  border-radius: 10px;
+  background-color: #B3D4FC;
+  background-image: -webkit-gradient(linear,
+									   left bottom,
+									   left top,
```

Comment:
looks clean just the indentation problem
Can be merge tho will resolve this issue after i add eslint stylelint husky and storybook

Reply:
should I fix the indentation?

Reply:
Yes fix it

Reply:
Done :) @ArunTeltia